ext/node/errors.rs

```diff
@@ -61,13 +63,26 @@ pub fn err_invalid_package_target(
 ) -> AnyError {
   let rel_error = !is_import && !target.is_empty() && !target.starts_with("./");
   let mut msg = "[ERR_INVALID_PACKAGE_TARGET]".to_string();
+  let pkg_json_path = PathBuf::from(pkg_path).join("package.json");
```

Comment:
@dsherret judging from this line, I'm thinking it should address the `deno test` error I referenced here https://github.com/denoland/deno/issues/16653#issuecomment-1355167325

Reply:
Yup it will. The error you got was what it used to give with `deno check` or `deno test` (since `deno test` does type checking).